saleor/graphql/payment/mutations.py

```diff
@@ -675,12 +735,26 @@ def validate_metadata_keys(cls, metadata_list: List[dict], field_name, error_cod
             raise ValidationError(
                 {
                     "transaction": ValidationError(
-                        {
-                            field_name: ValidationError(
-                                "Metadata key cannot be empty.",
-                                code=error_code,
-                            )
-                        }
+                        f"{field_name} key cannot be empty.",
+                        code=error_code,
+                    )
+                }
+            )
+
+    @classmethod
+    def validate_external_url(
+        cls, external_url: Optional[str], parent_field_name: str, error_code: str
+    ):
+        if external_url is None:
```

Comment:
If the `external_url` will be an empty string we will raise an error, is it expected?

Reply:
I would say that this is expected. Like, when `externalUrl` is None, it means that input doesn't have this value or it was provided as a `null`, but empty string is a just incorrect value provided by user. 
Do you agree?

Reply:
We agreed that from frontend perspective it is better to recieve, `null` or fully valid `url`.